spec/requests/api/v1/add_partners_requests_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+describe "Partners API Requests", type: :request do
+  describe "POST /api/v1/add_partners" do
+    context "when given valid parameters" do
+      it "creates a new `Partner` record" do
```

Comment:
Should we also add a test for the "resends the invite" code path that this request could theoretically take?  (Full disclosure, Im not :100: sure what that would look like off the top of my head, but it feels like it might be worth doing.)

Reply:
I added a second test to show that when the user is already in the system it doesn't change the count. I don't know about testing the sending of the invite -- we can do that but it feels like testing devise?

Reply:
Yeah, I think that the test you added is enough. Just want to cover both halves of the conditional in some way.